src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+* Introduced a new metric, `http.server.request.duration`, for users who opt
+  into the new semantic convention by configuring the `OTEL_SEMCONV_STABILITY_OPT_IN`
+  environment variable. This metric measures time in seconds and offers unique
+  histogram buckets as per the
```

Comment:
> This metric measures time in seconds and offers unique
>   histogram buckets

Not sure how to interpret "offers unique histogram buckets"? What does it mean?

Reply:
"offers" is the wrong word here, "uses" would be more appropriate.
I'm trying to call attention to the fact that the buckets are different than the default.
4 lines below this I list the exact buckets.


I'm trying to find a succinct way to describe that this metric is different than what came before.
@cijothomas, do you think this would be better as a bulleted list?

ie:
```
- new metric: `http.server.request.duration`
- unit: seconds
- buckets: 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1,
  2.5, 5, 7.5, 10
```

Reply:
the list is good. Hint that the buckets are so, because SDK specially handles this metric.